builtins/amp-img.js

```diff
@@ -178,6 +202,22 @@ export class AmpImg extends BaseElement {
     return promise;
   }
 
+  /**
+   * Sets the img src to the first url in the srcset if srcset is defined but
+   * src is not.
+   * @private
+   */
+  guaranteeSrcForSrcsetUnsupportedBrowsers_() {
+    if (!this.img_.hasAttribute('src') && 'srcset' in this.img_ == false) {
+      const srcset = this.element.getAttribute('srcset');
+
+      const srcseturl = /\S+/.match(srcset);
```

Comment:
nice simplification. LGTM, can't think of cases this may fail, @jridgewell WDYT?

Reply:
Yah, this'll work.